Tasks/AzureIoTEdge/task.json

```diff
@@ -0,0 +1,244 @@
+{
+  "id": "80F3F6A0-82A6-4A22-BA7A-E5B8C541B9B8",
+  "name": "AzureIoTEdge",
+  "friendlyName": "Azure IoT Edge",
+  "description": "Azure IoT Edge",
+  "helpMarkDown": "",
+  "category": "Build",
+  "visibility": [
+    "Build",
+    "Release"
+  ],
+  "author": "Microsoft",
+  "version": {
```

Comment:
we should add "preview":true

Reply:
we should let the second version be in preview for a while to get feedback

Reply:
added preview tag